std_msgs/Header header
string topic_name
string topic_arg
float32 resolution
float32[] shape
float32[] position
float32[] index
float32 arg_at_position
float32[] topic_map
